data/json/martialarts.json

```diff
@@ -21,6 +23,7 @@
     "id": "style_brawling",
     "name": "Brawling",
     "description": "You're used to hand-to-creature fighting.  Not stylish or sporting, but it gets the job done.",
+	"initiate": ["You adopt a classic karate stance.", "%s adopts a classic karate stance."],
```

Comment:
This doesn't look like the correct text for Brawling

Reply:
Whoops! I must have missed that. I've changed it to the correct messaging.